File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        freq = Counter(nums)
        maxi = 0
        res = 0
        for v in freq.values():
            maxi = max(maxi, v)
        for v in freq.values():
            if v == maxi:
                res += v
        return res

File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq = new int[101];
        int max = 0; //maximum freq
        
        for(int i : nums) {
            freq[i]++;
            if(freq[i]>max) max = freq[i];
        }
        
        int ans = 0;
        
        for(int i : nums) {
            if(freq[i] == max) { 
                ans++;
            }
        }
        
        return ans;
    }
}